blocks/bricks/parallel.py

```diff
@@ -200,13 +208,18 @@ class Distribute(Fork):
         The names of the targets.
     source_name : str
         The name of the source.
+    target_dims : list
+        A list of target dimensions, corresponding to `target_names`.
+    source_dim : int
+        The dimension of the source input.
+    prototype : :class:`~blocks.bricks.Feedforward`, optional
+        The transformation prototype. A copy will be created for every
+        input. By default a linear transformation is used.
```

Comment:
I like the idea to rename the current `Linear` into `Affine`! Can you please create a ticket?